### DISCUSSION TITLE
Wildcard for citekey

### DISCUSSION THREAD

**Discussion by tomhaley** [retorquere/zotero-better-bibtex#2581] [Q&A] [answered]
Is there a wildcard or some other method that will let Zotero access the citekey for a given item when renaming an attachment? Zotero 7 supports automatic renaming of attachments on import, something I previously used Zotfile to do. As far as I can tell the feature is undocumented and changes to the rename format can only be done in the config editor. My preference is to name PDF attachments with the citekey but can't figure out a way to have Zotero use the citekey when doing the rename. If it helps, this is the applicable entry in the config editor: 
<img width="600" alt="image" src="https://github.com/retorquere/zotero-better-bibtex/assets/14948834/3c155d3f-1387-4383-a64f-b2331ce13f79">

**Comment by retorquere** [comment, marked as answer]
It isn't currently possible, but you can open a feature-request issue, and I will look into it. I don't know when I will be able to, I'm working in a few bugs ATM and I am deep into the Zotero 7 port.

**Comment by tomhaley** [comment]
Created an issue but definitely low priority, thanks!
